src/controllers/login.js

```diff
@@ -0,0 +1,8 @@
+exports.get = (request, response) => {
+  response.render('login', {
+    js: ['helpers/collectData', 'login'],
+    css: ['login'],
+    layout: 'login_signup',
+    title: 'Login | Gaza Freelancing Community',
```

Comment:
@Angham116 One thing that might be useful is to pull out this title into a function in another file, so that you can use it for all of your controllers:
```
const makeTitle = ( pageTitle ) => `${ pageTitle } | Gaza Freelancing Community`
```
.. and then use it here like this: `title: makeTitle( 'Login' )`
.. and then reuse it on other pages: `.. title: makeTitle( 'Signup' )`, etc.

This would make life easier if the page title format ever changed (which they often do, since it affects SEO a lot), for example if you wanted it to change to 'Login | Gaza Freelancing Community, find friendly freelancers, job postings', then you could just change one function instead of changing all of your controllers.